### PR TITLE
Fix workflow output

### PR DESCRIPTION
**Pull request by colinking**
## Description

Fixes an issue that prevented workflows from producing output from return values.

## Test plan

Run a workflow locally that returns a result as output. The output should be rendered in the UI.

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #204** <a href="https://app.graphite.dev/github/pr/airplanedev/lib/204" target="_blank">Graphite</a>  👈
    * **PR #203** <a href="https://app.graphite.dev/github/pr/airplanedev/lib/203" target="_blank">Graphite</a> 
      * **PR #202** <a href="https://app.graphite.dev/github/pr/airplanedev/lib/202" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/lib/204?utm_source=stack-comment).

File: pkg/build/workflow/workflow-shim.js [hunk 1]
```diff
@@ -17,16 +17,14 @@ export async function __airplaneEntrypoint(params, workflowArgs) {
     }
 
     var result = await task(JSON.parse(params[0]));
```
Comment (colinking) [inline pkg/build/workflow/workflow-shim.js:19]: This is the bug ^

`result` was scoped to the `try` so `undefined` was always returned as the workflow's output.

File: pkg/build/workflow/workflow-shim.js [hunk 2]
```diff
@@ -17,16 +17,14 @@ export async function __airplaneEntrypoint(params, workflowArgs) {
...
     logger.info('airplane_output_append:error ' + JSON.stringify({ error: String(err) }));
     logger.info('airplane_output_append:error {"error":"Error executing workflow"}')
-    return
-  }
-
-  if (result !== undefined) {
-    airplane.setOutput(result);
   }
-  logger.info('airplane_status:succeeded');
-  return result;
```
Comment (colinking) [inline pkg/build/workflow/workflow-shim.js:31]: There's no need to return the result AFAIK, so I removed it.

File: pkg/build/workflow/worker-shim.js [hunk 3]
```diff
@@ -113,26 +112,17 @@ function activityLog(info, message) {
...
+    console.error(`Expected to receive a single argument (via {{ "{{JSON}}" }}). Task CLI arguments may be misconfigured.`);
     process.exit(1);
   }
 
   try {
-    let ret = await runWorker(JSON.parse(process.argv[2]));
-    if (ret !== undefined) {
-      airplane.setOutput(ret);
-    }
+    await runWorker(JSON.parse(process.argv[2]));
```
Comment (colinking) [inline pkg/build/workflow/worker-shim.js:123]: Pretty sure this is a relic of an old version of the worker, but the worker itself does not produce output.

**Review by calufornia — APPROVED** [review]
lgtm - thanks for cleaning up the logs too!

**Pull request merged**